src/main/java/com/_up/megastore/services/implementations/UserService.java

```diff
@@ -127,7 +128,7 @@ private User findUserToActivateOrThrowException(UUID activationTokenUUID) {
   }
   private void ifUserIsActivatedThrowException(User user){
     if(user.isActivated()){
-      throw new IllegalArgumentException("User with UUID " + user.getUserId() + " is already activated.");
```

Comment:
Por qué se sacó el id del log? 🤔

Reply:
Porque esto se le muestra al usuario. Supongamos que el usuario ingresa un mail ya activado, en el toast aparece 'User with id ... is already activated'